build.gradle

```diff
@@ -29,7 +29,7 @@ buildscript {
 plugins {
   id "com.github.sherter.google-java-format" version "0.8"
   id "net.ltgt.errorprone" version "0.7" apply false
-  id "com.github.johnrengelman.shadow" version "2.0.4" apply false
+  id "com.github.johnrengelman.shadow" version "6.1.0" apply false
```

Comment:
Did we mean to bump this dep in this diff?  Either way is ok with me

Reply:
Nope, was already planning to change this back. That will be on the other PR.

Reply:
Actually, lets just land #461 first. Then this will auto-magically fix upon rebase, I think.